ruby/red-arrow/lib/arrow/table.rb

```diff
@@ -287,6 +318,8 @@ def slice(*args)
       filter_options.null_selection_behavior = :emit_null
       sliced_tables = []
       slicers.each do |slicer|
+        filter_options.null_selection_behavior = :drop if slicer == Arrow::Slicer::GreaterEqualCondition
+
```

Comment:
Could you revert this for now?
This breaks existing tests.

How about working on this in another pull request?

BTW, `slicer == Arrow::Slicer::GreaterEqualCondition` is never true because `slicer` is a condition object not condition class.

Reply:
Oh, I forget to delete debug code...
I removed this code!